src/main/resources/assets/thehallow/textures/block/rainbow_pumpkin_side.png.mcmeta

```diff
@@ -0,0 +1,20 @@
+{
+   "animation": {
+    "interpolation": true,
+    "frametime": 2,
+    "frames": [
+      0,
+      0,
+      2,
+      2,
+      1,
+      1,
+      5,
+      5,
+      4,
+      4,
+      3,
+      3
```

Comment:
Why 0 2 1 5 4 3? Why not just reorder the texture to have the frames in order?

Reply:
Too lazy tbh. Also the ingot is similar but makes sense. Looks less sezuire like in this order, but really every order is probably seizure inducing.